types/aframe/tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
The only tests the CI run are definition tests. no functional tests. the definition tests assert that the change to the declaration file does not cause errors in valid usage of the API.